src/interfaces/IPaprController.sol

```diff
@@ -238,23 +243,21 @@ interface IPaprController {
     /// @param collateral address of the collateral token
     function isAllowed(ERC721 collateral) external view returns (bool);
 
+    /// @notice returns the timestamp at which an asset was proposed to be added as allowed collateral
+    /// @param asset address of the ERC721 token
+    function proposedTimestamp(ERC721 asset) external view returns (uint256);
+
     /// @notice if liquidations are currently locked, meaning startLiquidationAuciton will revert
     /// @dev for use in case of emergencies
     /// @return liquidationsLocked whether liquidations are locked
     function liquidationsLocked() external view returns (bool);
 
-    /// @notice boolean indicating whether token0 in pool is the underlying token
-    function token0IsUnderlying() external view returns (bool);
```

Comment:
probably need a subgraph update with this, heads up

Reply:
Yup thanks!